### PR TITLE
Move section about undefined this inside modules

### PR DESCRIPTION
**Pull request by leo-diehl**
Move the section about "undefined global this" within the "inbuilt strict mode" one.
This characteristic is not specific to modules but related to the inbuilt use-strict that modules have,
and the way the information was placed could mislead the user to think that it was module-exclusive.

### CODE REVIEW

**Comment by joaquinelio** [comment]
I wouldnt delete it...
maybe rephrase it  adding the why 

the point is the difference,  window obj  (even with strict) vs undefined.

I guess no-strict outside the browser it should return system obj

**Comment by iliakan** [comment]
@leo-diehl please elaborate.

Looks like you think that 'use strict' implies undefined `this`. It's not so. Modules do that.

Please check this example: http://plnkr.co/edit/caiRRgRbfqsc84Yr?open=lib%2Fscript.js

**Comment by leo-diehl** [comment]
@iliakan, thanks for the code example, it gave me an insight that I didn't have had before: that the functions created inside a strict environment have their `this` undefined, not the environment itself.

I would like to keep my suggestion, please check this example:
http://plnkr.co/edit/4XvG7xorHd3CUf8T?open=lib%2Fscript.js&preview

**Comment by iliakan** [comment]
These are different rules.

1. A function in `use strict` mode has `this=undefined`.
2. Modules in global scope has `this=undefined`.
3.

**Comment by leo-diehl** [comment]
Thanks for the clarification!
